src/components/Library/MuseumSection.js

```
@@ -0,0 +1,40 @@
+import React from 'react';
+import { View, Text, FlatList, StyleSheet } from 'react-native';
+import ArtworkCard from './ArtworkCard';
+
+const MuseumSection = ({ title, artworks, onPressArtwork }) => (
+  <View style={styles.section}>
+    <Text style={styles.title}>{title}</Text>
+    <FlatList
+      data={artworks}
+      renderItem={({ item }) => (
+        <ArtworkCard {...item} onPress={() => onPressArtwork && onPressArtwork(item.id)} />
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)